src/fiatExchanges/quotes/FiatConnectQuote.test.ts

```diff
@@ -22,6 +22,20 @@ jest.mock('src/web3/contracts', () => ({
     getAccounts: jest.fn(() => ['fake-account']),
   })),
 }))
+jest.mock('src/fiatExchanges/quotes/constants', () => {
+  const originalModule = jest.requireActual('src/fiatExchanges/quotes/constants')
+
+  return {
+    ...originalModule,
+    DEFAULT_ALLOWED_VALUES: {
+      // Using AccountNumber because jest hoisting prevents us from using the
+      // FiatAccountSchema enum.
```

Comment:
fwiw I'm usually fine with hardcoding things in test code since it keeps it more independent from the rest of the codebase, and less circular. this can actually help prevent bugs-- for instance, if you updated some constant to a buggy value, and then re-used that constant in test code, you'd be in trouble. But if the test code has some hardcoded version of it that's supposed to match, it's less likely that you made the same typo twice, and you might notice the new discrepancy from a failing test